python/tvm/relay/op/contrib/ethosu.py

```diff
@@ -1810,27 +1811,37 @@ def __init__(self, func_body):
         from tvm.relay.backend.contrib.ethosu.util import QDenseArgs  # type: ignore
         from tvm.relay.backend.contrib.ethosu.util import BiasAddArgs, RequantArgs
 
+        fract_scale = tvm.relay.Constant(tvm.nd.array(np.array(1 / 2**15)))
```

Comment:
Out of curiosity, where does `1/2**15` come from?

Reply:
It was a certain case where 15 bits were used to the fractional part, now this value is parameterized.